js_modules/dagit/packages/core/src/workspace/WorkspaceAssetsRoot.tsx

```diff
@@ -27,6 +30,7 @@ export const WorkspaceAssetsRoot = ({repoAddress}: {repoAddress: RepoAddress}) =
     },
   );
   const {data, loading} = queryResultOverview;
+  const refreshState = useQueryRefreshAtInterval(queryResultOverview, POLL_INTERVAL);
```

Comment:
If you want, I think we have the standard 15s default defined inside `QueryRefresh.tsx` and we could import that one. I moved it there during a refactor because it seemed like it should be the same in most cases.